cyberbattle/_env/cyberbattle_env.py

```diff
@@ -238,10 +238,10 @@ class AttackerGoal(NamedTuple):
     # Include goal to bring the availability to lower that the specified SLA value
     low_availability: float = 1.0
     # Include goal to own at least the specified number of nodes.
-    own_atleast: int = 0
+    own_atleast: int = 0.0
     # Include goal to own at least the specified percentage of the network nodes.
     # Set to 1.0 to define goal as the ownership of all network nodes.
-    own_atleast_percent: float = 1.0
+    own_atleast_percent: float = 0.0
```

Comment:
That would not be a valid goal: it's immediately met at the start of the simulation. Also changing the default value can break people. Why not just setting the parameter when instantiating your environment?

Reply:
In this case the default value zero percentage will never be reached due to the strict sign of condition here https://github.com/kvas7andy/CyberBattleSim/blob/benchmark_issues/cyberbattle/_env/cyberbattle_env.py#L1049. To be safe it is possible to use negative value as default.

It definitely can break people's code. But I think I have forgot to include one more change in this  PR, which updates the default environments parameters. Thus it's on your behalf, do you want to update old env version "v0" in order to preserve same conditions and make new versions "v1" or the other way around. I have made several comments about it in **Issue 2** of #87. 

Let me include that changed file.

Because that changes the perspective of what is default goal setting, I believe it would be better to change parameter value for two reasons. 1) if not changed, default goal will always be to at least own the whole network which is not the case for TinyToy and ToyCTF goals from their description you made in Readme. 2) I would like to see visually which goals are included by setting the value when instantiating AttackerGaol object, because from the code I figured out, all this parameters are set in a multiplication conditioning. So visually it will have the same logic, if I set parameters value different from default, I want to include this condition. While now, without setting own_atleast_percent we by default include this condition, because of value equal to 1.0

Reply:
New change in [file](https://github.com/microsoft/CyberBattleSim/blob/be408777d9ba5b285c7cc3c84da690eb068a9433/cyberbattle/__init__.py) to preserve same behavior as before for environments with v0.